### PR TITLE
SAIC-675 Invalid flavorRef while running checks.

### PR DESCRIPTION
**Pull request by plomakin**
Fully reworked check_cloud action.

### CODE REVIEW

File: cloudferrylib/os/actions/check_cloud.py [hunk 1]
```diff
@@ -152,20 +172,61 @@ def run(self, **kwargs):
...
+            'display_name': 'test_volume',
+            'volume_type': None
+        }
+
+        try:
+            with self.create_tenant(ks_client, tenant_name), \
+                    self.create_flavor(nv_client, flavor), \
+                    self.create_image(gl_client, image_info):
+
+                tenant_id = ks_client.get_tenant_id_by_name(tenant_name)
```
Comment (plomakin) [inline cloudferrylib/os/actions/check_cloud.py]: You're right. Will fix it.

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.check_cloud
cloudferrylib/os/actions/check_cloud.py:40: [W1202(logging-format-interpolation), CheckCloud.create_tenant] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:50: [W1202(logging-format-interpolation), CheckCloud.create_flavor] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:60: [W1202(logging-format-interpolation), CheckCloud.create_network] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:66: [W1202(logging-format-interpolation), CheckCloud.create_network] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:72: [W1202(logging-format-interpolation), CheckCloud.create_subnet] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:78: [W1202(logging-format-interpolation), CheckCloud.create_subnet] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:84: [W1202(logging-format-interpolation), CheckCloud.create_image] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:89: [W1202(logging-format-interpolation), CheckCloud.create_image] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:94: [W1202(logging-format-interpolation), CheckCloud.create_volume] Use % formatting in logging functions but pass the % parameters as arguments
cloudferrylib/os/actions/check_cloud.py:99: [W1202(logging-format-interpolation), CheckCloud.create_volume] Use % formatting in logging functions but pass the % parameters as arguments

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
Looks good, +1

**Comment by archyufa** [comment]
Failing on staging.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by archyufa** [comment]
Tested in Staging works +1

**Comment by roman-verchikov** [comment]
+1

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by plomakin** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**